readme.md

```diff
@@ -100,10 +100,10 @@ Getting the code with the test dependencies (`-t` switch):
 
 Build and test all packages:
 
-  cd src/github.com/zalando/skipper
-  make deps
-  make install
-  make shortcheck
+    cd src/github.com/zalando/skipper
+    make deps
+    make install
+    make shortcheck
```

Comment:
please drop these changes

Reply:
But it makes it look better:

<img width="1858" alt="default" src="https://user-images.githubusercontent.com/189443/45054371-0b399300-b08d-11e8-9205-563557e80a22.png">

Reply:
Oh, I did not know, that it's not correctly rendered right now. :)